app/javascript/packs/dashboard.js

```diff
@@ -1,5 +1,11 @@
+import { widget as dashboardWidget } from '../src/Dashboard/index'
+import { initialize as toggleWidget } from '../src/Dashboard/toggle'
+import { render as renderChartWidget } from '../src/Dashboard/chart'
 import { ApiFilterWrapper } from '../src/Dashboard/components/ApiFilter'
 
 document.addEventListener('DOMContentLoaded', () => {
+  window.dashboardWidget = dashboardWidget
```

Comment:
Do we really need to use globals? There is no way to load ES6 modules?

Reply:
In many places we could use `content_tag`: https://github.com/rails/webpacker/blob/3-x-stable/docs/props.md#react

Reply:
I'll open an issue for this, in order to apply this where it's possible